src/components/LeafletMarker/index.js

```diff
@@ -43,7 +42,15 @@ const LMarker = (props) => {
 };
 
 LMarker.propTypes = {
-  device: PropTypes.object,
+  device: PropTypes.shape({
+    latitude: PropTypes.number,
+    longitude: PropTypes.number,
+    types: PropTypes.arrayOf({
```

Comment:
This is not a correct construct. It should be either
```jsx
types: PropTypes.arrayOf(PropTypes.string)
```
or
```jsx
types: PropTypes.shape({
  name: PropTypes.string,
})
```

Reply:
I don't get that, types is an array of objects with the name property right?
`
[
{name:"foo"},
{name:"bar"},
]
`

Reply:
Then it should be the latter:
```jsx
types: PropTypes.shape({
  name: PropTypes.string.isRequired,
})
```

Reply:
Using that construct throws errors. I updated the shape so it doesn't throw errors and works